### DISCUSSION TITLE
Reading Petroncini TT 5/10

### DISCUSSION THREAD

**Discussion by Drahos00** [artisan-roaster-scope/artisan#583] [Q&A]
Hello,

im trying to configure Artisan with Petroncini TT 5/10 (2020) equipped with the Avantgarde control system talking Siemens S7.

1. In a general im unable to see the roaster on local network using address in the range of the roaster (192.168.100.x).
2. The roaster have default IP adress: 192.168.100.2 (Instead of 192.168.100.1 how is described in setup guide on https://artisan-scope.org/machines/petroncini/).
3. When i use ping command on device (PC with Linux OS) i have installed Artisan im expecting some respons from adress of the roaster but result is just "Destination host unreachable"
4. I also tried used PC with Windows 10 but with same result

So before Configuring Artisan i would like get / check connectivity between devices.
Roaster operate on Windows CE where im not sure if threre is some update or drivers instalation needed.

So i would like to ask you about any help / tips what should i try and check.

Thank you very much for any respons.

**Comment by MAKOMO** [comment]
An S7 PLC should answer a ping. Did you put your PC (via a manual config) on the same local network as the roasters IP (thus on 192.168.100.x)? If you cannot ping the PLC from your computer it is unlikely that Artisan can connect to it.

**Reply by Drahos00** [reply]
Yes i setup static IPv4 adress (192.168.100.42) on my Win PC (also tryed same settings on Linux PC via profiles) and Roaster is still not comunicate with me on any PC. 

May we have some problem with LAN Card in PLC or Internet connection setting there. But everythink seems actually adjusted according manuals.

So we also contacted dealer of Petroncini roaster with this Issue.

**Reply by MAKOMO** [reply]
Any news on this? Are you sure that the roasters IP address is 192.168.100.2 and not 192.168.100.1?
